lib/postgres_ext/serializers/active_model/array_serializer.rb

```diff
@@ -51,7 +62,9 @@ def _include_relation_in_root(relation, *args)
       end
 
       _serializer = serializer_class.new klass.new, options
-      root_key = local_options.fetch(:root, _serializer.root_name.to_s.pluralize).to_s
```

Comment:
This code was buggy, because root option could be nil, causing the fallback to not be used.